glean-core/python/glean/_ffi.py

```diff
@@ -98,6 +98,13 @@ def ffi_encode_vec_string(strings: List[str]) -> Any:
     return result
 
 
+def ffi_encode_vec_int32(values: List[str]) -> Any:
```

Comment:
`List[str]`? Shouldn't this be `List[int]`?

Reply:
Good catch.